platform/platform-logging/src/main/java/org/codice/ddf/logging/platform/LogEvent.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.logging.platform;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+/**
+ * Describes a log event in the system
+ */
+public class LogEvent implements Comparable<LogEvent> {
+
+    private final long timestamp;
+
+    private final String level;
+
+    private final String message;
+
+    private final String bundleName;
+
+    private final String bundleVersion;
+
+    /**
+     * @param timestamp
+     *            timestamp of this {@link LogEvent}
+     * @param level
+     *            level of this {@link LogEvent}
+     * @param message
+     *            log message of this {@link LogEvent}
+     * @param bundleName
+     *            the name of the bundle creating this {@link LogEvent}
+     * @param bundleVersion
+     *            the version of the bundle creating this {@link LogEvent}
+     */
+    public LogEvent(long timestamp, String level, String message, String bundleName,
```

Comment:
Could we make this constructor package-private and take in a `PaxLoggingEvent` instead of all the individual pieces? That way the class could keep a reference to the `PaxLoggingEvent` and have the getters simply delegate to it.